src/custom-sw-code.js

```diff
@@ -24,11 +24,9 @@ const catchHandler = async ({ event }) => {
 
   if (dest === 'document') {
```

Comment:
Just for me to understand, why were these changes necessary in this file?

Reply:
sonarqube problem